CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+## Contributing Guide
+
+### Getting Started
+
+This guide will help you setup and contribute to jovian.
+
+**Step 1: Clone the repo:**
```

Comment:
Can remove the colon for all the steps and can use `repository` instead of `repo`.